discord/appinfo.py

```diff
@@ -52,9 +55,13 @@ class AppInfo:
         grant flow to join.
     rpc_origins: Optional[List[:class:`str`]]
         A list of RPC origin URLs, if RPC is enabled.
+    game_info: Optional[:class:GameInfo]
```

Comment:
```suggestion
    game_info: Optional[:class:`GameInfo`]
```

Reply:
Since we are removing GameInfo, I'm resolving this comment.